.github/workflows/docs.yml

```diff
@@ -84,5 +84,9 @@ jobs:
           ```
 
           Alternatively, wait for this branch to be out-of-date with master, then just use the "Update branch" button!
+          
+          Note that it is expected that the PennyLane-Lightning repo is a version ahead of the release, because the 
+          version number is taken from the dev branch. Trying to `pip install` from the files will fail until that 
+          major version of Lightning is released.
```

Comment:
What would be the recommended alternative?

Reply:
My understanding was that this is mostly for 'going back in time', so usually, we if we want to use this, the major version of lightning that was the dev version when this file was updated _would_ be released. So you could just pip install from the file.

If I wanted to go back to the very recent past, I would just manually update the files locally to remove the line about lightning before pip installing from the file, and install the appropriate dev version based on what I was troubleshooting, but maybe there's a better way.

@timmysilv, is that a fair summary?

Reply:
I think so. In short:
- you probably don't care about the specific lightning version, so downgrade by 1 if `pip install` failed with not found error
- if you _do_ want the specific dev version, remove that line and install it from local manually because you're a hacker and you know what you need

I imagine the latter group will have deduced as much from the text that's already added, so I'd focus on the former and just suggest downgrading the version by one in the file to get something stable.